code/modules/mob/living/carbon/human/species_types/vox.dm

```diff
@@ -27,7 +27,8 @@
 	burnmod = 0.8 // Tough hides.
 	stunmod = 1.1 // Take a bit longer to get up than other species.
 	breathid = "n2"
-	examine_text = "A vox outcast"
+	examine_text = "A Vox Outcast"
```

Comment:
Shouldn't it be `A Vox outcast`?

Reply:
The other ones have a as a lower case though 🤔